docs/reference/mapping/dynamic/field-mapping.asciidoc

```diff
@@ -21,7 +21,7 @@ h| JSON data type h| `"dynamic":"true"` h| `"dynamic":"runtime"`
  |`null` 2*| No field added
  |`true` or `false` 2*| `boolean`
  |`double` | `float` | `double`
- |`integer` 2*| `long`
+ |`long` 2*| `long`
```

Comment:
I also think clarifying that the first column is JavaScript parser column and the second two columns are ES types would be useful.